packages/js/components/src/split-dropdown/test/index.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+import { createElement } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { SplitDropdown } from '..';
+
+describe( 'SplitDropdown', () => {
+	test( 'it renders correctly collapsed', () => {
+		const { container, getByText, queryByText } = render(
+			<SplitDropdown>
+				<Button>Default Action</Button>
+				<Button className="test-collapsed-button">
+					Secondary Action
+				</Button>
+				<Button className="test-collapsed-button">
+					Tertiary Action
+				</Button>
+			</SplitDropdown>
+		);
+		expect( container ).toMatchSnapshot();
```

Comment:
I'm not a big fan of snapshots since they tend to be brittle if the underlying component structure changes.  This could happen outside of our control too and puts the onus on the next person updating to re-review logic and hopefully not miss anything.